ONSdigital/alpha-eq-survey-runner
# DIGEQ-94 Fix for too many open files

**What**
During load testing the application was throwing too many open files errors. This resulted from it opening a new cassandra session on every request. I've change it so it opens a single cassandra session for application, as the session object automatically performs connection pooling.

**How to test**
1. Checkout this branch
2. Run the debug survey
2. Using multiple cassandra sessions check you can perform 

**Who can review**
Anyone apart from @warren-methods
